docs/reference/compatibility.txt

```diff
@@ -491,3 +491,58 @@ are supported by Mongoid.
   8.0 and 8.1 stable branches.
 
 .. include:: /includes/unicode-checkmark.rst
+.. include:: /includes/unicode-ballot-x.rst
+
+Rails Frameworks Support
+------------------------
+
+Ruby on Rails is comprised of a number of frameworks, which Mongoid attempts to
+provided compatibility with wherever possible.
+
+.. list-table::
+   :header-rows: 1
+   :stub-columns: 1
+   :class: compatibility-large no-padding
+
+   * - Rails Framework
+     - Supported?
+
+   * - ``ActionCable``
+     - |checkmark| [#rails-actioncable-dependency]_
+
+   * - ``ActionMailbox``
+     - |x| [#rails-activerecord-dependency]_
+
+   * - ``ActionMailer``
+     - |checkmark|
+
+   * - ``ActionPack``
+     - |checkmark|
+
+   * - ``ActionText``
+     - |x| [#rails-activerecord-dependency]_
+
+   * - ``ActionView``
+     - |checkmark|
+
+   * - ``ActiveJob``
+     - |checkmark| [#rails-activejob-dependency]_
+
+   * - ``ActiveModel``
+     - |checkmark|
+
+   * - ``ActiveRecord``
+     - |x|
+
+   * - ``ActiveStorage``
+     - |x| [#rails-activerecord-dependency]_
+
+.. [#rails-activerecord-dependency] Depends on ``ActiveRecord``
+
+.. [#rails-activejob-dependency] Serialization of BSON & Mongoid objects is
+   fraught, works best if you explicitly send ``ObjectId``'s as strings, and
+   reconstitute them in the job
+
+.. [#rails-actioncable-dependency] not all subscription adapters are compatible
+   with Mongoid (e.g. the PostgreSQL adapter) but most if not all other adapters
+   are compatible.
```

Comment:
Looking for feedback on how best to phrase these :)

Reply:
Feels like the activejob footnote might be best given as an example somewhere, e.g.

```ruby
record = Model.find(...)
MyJob.perform_later(record._id.to_s)

class MyJob < ApplicationJob
  def perform(id_as_string)
     id = BSON::ObjectId.from_string(id_as_string)
     record = Model.find(id)
     # ...
  end
end
```

For the ActionCable footnote, I'm wondering now if that's even necessary to say. I can't imagine anyone would expect the PostgreSQL adapter to work with MongoDB. Could perhaps just say something like, "There is no MongoDB adapter for ActionCable. Redis is a robust backend for ActionCable, and we recommend using it instead." As Basecamp/37signals, we used Redis extensively, despite using MySQL for all of our persistent data. Seems like a pretty common pattern.